src/main/java/com/dougnoel/sentinel/filemanagers/FileManager.java

```diff
@@ -98,4 +104,60 @@ public static String getClassPath(String className) {
 			return null;
 		}
 	}
+	
+	/**
+	 * Saves an image of type File to the given sub-directory of the configured, or default, image path.
+	 * 
+	 * @param subDirectory the sub-directory within the configured or default image path.
+	 * If the sub-directory is null, the root image directory will be used.
+	 * @param imageFileName the file name of the image to save.
+	 * @param imageFile the File of the image to save.
+	 */
+	public static void saveImage(String subDirectory, String imageFileName, File imageFile) throws IOException {
+		if(subDirectory == null) 
+			subDirectory = "";
+		else
+			subDirectory += "/";
+		
+		File destinationFile = new File(Configuration.imageDirectory() + "/" + subDirectory + imageFileName);
+    	FileUtils.copyFile(imageFile, destinationFile);
+	}
+	
+	/**
+	 * Saves an image of type BufferedImage to the given sub-directory of the configured, or default, image path.
+	 * 
+	 * @param subDirectory the sub-directory within the configured or default image path.
+	 * If the subDirectory is null, the root image directory will be used.
+	 * @param imageFileName the file name of the image to save.
+	 * @param imageFile the BufferedImage of the image to save.
+	 */
+	public static void saveImage(String subDirectory, String imageFileName, BufferedImage imageFile) throws IOException {
+		if(subDirectory == null) 
+			subDirectory = "";
+		else
+			subDirectory += "/";
+		
+		File destinationFile = new File(Configuration.imageDirectory() + "/" + subDirectory + imageFileName);
+		//ImageIO will not create directories. We have to make them ourselves before running the write command.
+		FileUtils.forceMkdir(destinationFile.getParentFile());
+    	ImageIO.write(imageFile, "png", destinationFile);
+	}
+
+	/**
+	 * Reads an image from the sub-directory of the default or configured image directory.
+	 * 
+	 * @param sourceImageSubDirectory the sub-directory within the configured or default image path to read the file from.
+	 * If the sub-directory is null, the root image directory will be used.
```

Comment:
This line should go in the description above.

Reply:
Done